refnx/dataset/test/test_dataset.py

```diff
@@ -1,7 +1,8 @@
-import os.path
 from pathlib import Path
-import pytest
 import glob
```

Comment:
In `test_load_data` (below), pathlib's glob can be used id you want, then omitting this import. https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob